src/passport-saml/saml.ts

```diff
@@ -1333,7 +1359,7 @@ class SAML {
 
     const wrappedKey = [
       '-----BEGIN PRIVATE KEY-----',
-      ...key.match(/.{1,64}/g),
+      ...key.match(/.{1,64}/g)!,
```

Comment:
I looked up what the "!" was for here. It expresses that the value cannot be `null` or `undefined`. 

https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci

Reply:
yes, JS had a problem - it did not check if key.match(/.{1,64}/) actually finds something and returns a match. Current PR copies this behavior. Here is transpiled version:

![image](https://user-images.githubusercontent.com/75169/97747567-d8687980-1af4-11eb-86d2-3f7ace4c9369.png)

Fixed in 36263b2